Source/KSCrash/Source/KSCrash/Recording/BSG_KSCrash.h

```diff
@@ -137,7 +137,8 @@ typedef enum {
                    metadata:(NSDictionary *)metadata
                      config:(NSDictionary *)config
                discardDepth:(int)depth
-           terminateProgram:(BOOL)terminateProgram;
+           terminateProgram:(BOOL)terminateProgram
+                  eventJson:(NSDictionary *)eventJson;
```

Comment:
The "Json" suffix seems a little inaccurate here - can we call the parameter `eventOverrides` and move it next to `callbackOverrides`?

Also - the doc-comments above need updating.